routes/v1.js

```diff
@@ -163,18 +175,19 @@ api.declare({
         routingKey:  '#'
       }
```

Comment:
Specifically:
```yaml
  bindings:
    type:     array
    default:  []
    items:
      type:                 object
      properties:
        exchange:
          title:            "Exchange"
          type:             string
        routingKey:         {$const: routing}
      additionalProperties: false
      required:
        - exchange
        - routingKey
```

Also see that schemas needs `description` attributes... nit, and not critical yet, but we should write a few small essays in there before we encourage people to use it :)